CONTRIBUTING.md

```diff
@@ -1,81 +1,120 @@
-## Contribution Guide
+# Contribution Guide
 
-### Settting up your local development environment
+## Settting up your local development environment
 
-#### Add the wasm target
+### Add the wasm target
 
 ```bash
 rustup target add wasm32-unknown-unknown
 ```
 
-#### Build
+### Build
 
 ```bash
 cargo build --target wasm32-unknown-unknown
 ```
 
-#### Examples
+#### stdweb
 
-TODO: Add more info 
-
-##### stdweb
-In order to run the examples in `./yew-stdweb`, you may wish to install [cargo-web]:
+In order to run the examples in `./yew-stdweb`, you may wish to install [cargo-web](https://github.com/koute/cargo-web):
 
 ```bash
 cargo install cargo-web
 ```
 
-[cargo-web]: https://github.com/koute/cargo-web
+## Tests
 
-#### Test
+Yew has unit tests which can be run with `cargo test`. However, a large portion of the tests need to be run in a browser.
+This section will guide you through the process of running these tests locally.
 
-##### Web Tests
-First, ensure that `wasm-bindgen-cli` is installed.
-[Instructions](https://rustwasm.github.io/docs/wasm-bindgen/wasm-bindgen-test/usage.html#install-the-test-runner)
+### Integration Tests
 
-Additionally a webdriver must be installed locally and configured to be on the
-`PATH`. Currently supports `geckodriver`, `chromedriver`, and `safaridriver`,
-although more driver support may be added! You can download these at:
+First, ensure that `wasm-pack` is installed.
+[Instructions](https://rustwasm.github.io/wasm-pack/installer/)
 
-* geckodriver - https://github.com/mozilla/geckodriver/releases
-* chromedriver - http://chromedriver.chromium.org/downloads
-* safaridriver - should be preinstalled on OSX
+`wasm-pack` automatically takes care of installing the correct `wasm-bindgen` version and downloading a webdriver for the browser.
 
-Lastly, the fetch tests require a local httpbin server running on port `8000`. We recommend running this with Docker:
+The following command is all you need to run the tests:
 
 ```bash
-docker run -p 8000:80 kennethreitz/httpbin
+wasm-pack test --firefox --headless -- --features wasm_test
```

Comment:
Maybe useful to also enable some additional features? We might also want to enable them below?
```suggestion
wasm-pack test --firefox --headless -- --features wasm_test,yaml,msgpack,cbor,toml,doc_test
```

Reply:
`yaml,msgpack,cbor,toml` don't have any tests, they really only activate extra dependencies + some glue to include them in Yew.
`doc_test` doesn't seem to activate any additional features. It was once used in the `build.rs` file to avoid panicking:
https://github.com/yewstack/yew/blob/86af481c0ddac47013480fc85daf482559b244b4/build.rs#L24-L32

This has since been removed and now it's just a dead feature I think.

So in the interest of keeping the command short and memorable I think it's best not to include these features.